www/README.md

```diff
@@ -11,14 +11,14 @@ Source for [harp.gl](https://heremaps.github.io/harp.gl) "landing" page
 
 ## Production architecture
 
-Main part of "app" is hosted on [Heremaps org Github Pages](https://heremaps.github.io/harp.gl).
+Main part of "app" is hosted on [Heremaps org Github Pages](https://www.harp.gl).
 
 Examples and docs are hosted on S3, behind Cloudfront.
 
 (Looks like) Cloudfront is the frontend for `harp.gl`, works as proxy for:
```

Comment:
This `looks like` looks like we don't know what we do :)
Can you please remove it or clarify what is unclear.

Reply:
Fixed